mandatory/2-function-creation.js

```diff
@@ -15,26 +21,52 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 199) {
+    return totalPrice - (totalPrice / 100) * 10;
+  } else {
+    return totalPrice - (totalPrice / 100) * 5;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  for (var i = 0; i <= limit; i++) {
```

Comment:
Hi Nasir, well done! Great job, but I think we need to use a while loop for this exercise.

Reply:
thank you Irianni ,i will fix that right now.

Reply:
Also please use the `let` keyword for the `i` variable declaration